packages/elements/src/components/ino-autocomplete/ino-autocomplete.scss

```diff
@@ -9,8 +9,8 @@
   margin: 0.5rem 0 0 0;
   padding: 0;
   background-color: white;
-  box-shadow: 0 1px 5px #DBDBDB;
-  border-radius: 4px;
+  box-shadow: 0px 0px 22px 0px rgba(93, 91, 105, 0.35);
```

Comment:
We should use our theme as much as possible in our components. 
Can you try out if one of our neutral colors (with the same `alpha 0.35`) look good?